.circleci/config.yml

```diff
@@ -25,3 +25,33 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+
+      # Ensure that for every commit to master and for every
+      # release tag, there is an app version in the catalog.
+      - architect/push-to-app-catalog:
+          name: push-to-app-catalog-master
+          app_catalog: "conformance-catalog"
+          app_catalog_test: "conformance-test-catalog"
```

Comment:
I see now that this is the test catalog for the `conformance-catalog` but initially I assumed this was the "prod" catalog for `conformance-test`s. Not sure how to make this more clear, just thought I'd mention there might be some ambiguity with this name

Reply:
I am also wondering if this is a good idea - maybe we could use the `control-plane` catalog and add a `conformance` collection? That seems easier at least.
With this catalog, we also need to change tooling and so on :(

Reply:
I did a bit more _thinking_ and I think going for the collection is the way forward here.

Reply:
I don't understand the `catalog`/`collection` magic enough to weigh in on the tradeoffs there. It seems like introducing a catalog is something we should be able to do, so maybe this would be an opportunity to improve that, and it would be cool to have conformance-only CPs on-demand. But I also think the stuff in the `conformance` catalog would still need elements from the `control-plane` catalog (e.g. `release-operator`), so it would not be deployable individually anyway unless we duplicated those components.